### PR TITLE
1308: Release 2.4.0

### PR DESCRIPTION
**Pull request by benedeki**

### CODE REVIEW

File: _posts/2020-04-27-release-2.4.0.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Release v2.4.0
+tags: ["v2.4.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 27/04 2020 the new version 2.4.0 is out
```
Comment (Zejnilovic) [inline _posts/2020-04-27-release-2.4.0.md:8]: This date should correspond with the release date.
And file name with this post publish date

File: _posts/2020-04-27-release-2.4.0.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+<!--more-->
+
+## Standardization Improvements
+
+- [#1187]({{ site.github.issues_url }}/1187) If input data does not adhere to schema (complex type to primitive type) it's detected and either new error type is added to `errCol` of the defective row (default) or processing stopped with exception altogether (in case a new switch `--strict-schema-check` is set to `true`)
+
+    Parquet, and and any other future format without enforced schema, now behaves similarly as XML or others on a missing column - it's considered NULL
```
Comment (Zejnilovic) [inline _posts/2020-04-27-release-2.4.0.md:15]: Changes is a list and turns into \<ul> I am not sure a new line is good here.

File: _posts/2020-04-27-release-2.4.0.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- [#1285]({{ site.github.issues_url }}/1285) Validate source and raw checkpoints are present in **Standardization_
```
Comment (Zejnilovic) [inline _posts/2020-04-27-release-2.4.0.md:17]: ** Standardization ** 
** Instead of __

File: _posts/2020-04-27-release-2.4.0.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+## Standardization & Conformance Improvements
```
Comment (Zejnilovic) [inline _posts/2020-04-27-release-2.4.0.md:19]: Spark Jobs improvements?

Also I would group standardization, this and HyperConformance. They are the same module and each has one change.

File: _posts/2020-04-27-release-2.4.0.md (hunk 1 above)
Comment (benedeki) [reply _posts/2020-04-27-release-2.4.0.md:8]: I should still get it there in time (per UTC ;-) )

File: _posts/2020-04-27-release-2.4.0.md (hunk 4 above)
Comment (benedeki) [reply _posts/2020-04-27-release-2.4.0.md:19]: SparkJobs is the module name. But everywhere in the documentation we refer to each phase as **Standardization** and **Conformance**. And HyperConformance is another logical usage of the module.

File: _posts/2020-04-27-release-2.4.0.md (hunk 2 above)
Comment (benedeki) [reply _posts/2020-04-27-release-2.4.0.md:15]: Works but ugly. Will do it differently.

**Pull request merged**